bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/presentation-download-dropdown/component.jsx

```diff
@@ -82,30 +84,63 @@ class PresentationDownloadDropdown extends PureComponent {
 
     this.menuItems = [];
 
-    const toggleDownloadOriginalPresentation = (enableDownload) => {
-      handleToggleDownloadable(item);
+    const { filenameConverted, filename, downloadableExtension } = item;
+    const convertedFileExtension = filenameConverted?.split('.').slice(-1)[0];
+    const originalFileExtension = filename?.split('.').slice(-1)[0];
+    const changeDownloadOriginalOrConvertedPresentation = (enableDownload, isConverted) => {
+      let typeOfExport;
+      if (isConverted) {
+        typeOfExport = 'Converted';
+      } else {
+        typeOfExport = 'Original';
+      }
+      handleDownloadableChange(item, typeOfExport, enableDownload);
```

Comment:
```suggestion
    const changeDownloadOriginalOrConvertedPresentation = (fileState, enableDownload) => {
      handleDownloadableChange(item, fileState, enableDownload);
```

O think receiving the `typeOfExport` as parameter will simplify it.
I suggest to rename `typeOfExport` to `fileState` to make more clear what this information means.